src/sentry/static/sentry/app/components/organizations/headerItem.jsx

```diff
@@ -53,30 +58,58 @@ class HeaderItem extends React.Component {
         className={className}
         isOpen={isOpen}
         hasSelected={hasSelected}
+        locked={locked}
         {...props}
       >
-        <IconContainer hasSelected={hasSelected}>{icon}</IconContainer>
+        <IconContainer hasSelected={hasSelected} locked={locked}>
+          {icon}
+        </IconContainer>
         <Content>{children}</Content>
         {hasSelected &&
-          allowClear && <StyledClose src="icon-close" onClick={this.handleClear} />}
-        <StyledChevron
-          isOpen={isOpen}
-          hasChanges={hasChanges}
-          onClick={this.handleChevronClick}
-        >
-          <InlineSvg src="icon-chevron-down" />
-        </StyledChevron>
+          !locked &&
```

Comment:
nit: would be easier to read if we refactored these conditions to a variable e.g. `shouldShowCloseButton` and for the below icons as well maybe?